package.json

```diff
@@ -82,7 +82,7 @@
     "sinon": "1.17.3",
     "sinon-chai": "2.8.0",
     "style-loader": "0.13.1",
-    "webpack": "1.12.14",
+    "webpack": "1.14.0",
```

Comment:
I second getting this into master asap. @jebeck - what kind of testing would be needed to ensure this doesn't cause any regressions?